conanfile.py

```diff
@@ -0,0 +1,94 @@
+
+    def configure(self):
+        self.options["qt"].shared = self.options.shared
+
+    def requirements(self):
+        self.requires("qt/5.15.5")
+
+    def build_requirements(self):
+        if self.settings.os == "Windows":
+            self.tool_requires("jom/1.1.3")
+
+    def source(self):
+        self.run(f"git clone -b v{self.version} --recursive -- https://github.com/qbs/qbs {self.source_folder}")
+
+    def _get_library_pathes(self):
+        if self.settings.os == "Windows":
+            cmd = "echo %LD_LIBRARY_PATH%"
+            delimeter = ";"
+        elif self.settings.os == "Macos":
+            cmd = "; echo $DYLD_LIBRARY_PATH"
+            delimeter = ":"
+        else:
+            cmd = "echo $LD_LIBRARY_PATH"
+            delimeter = ":"
+
+        library_path_io = StringIO()
+        self.run(cmd, output=library_path_io, run_environment=True)
+        library_path_io.flush()
+        library_path = library_path_io.getvalue().strip().split(delimeter)
+        return " ".join(library_path)
+
+    def build(self):
+        library_path = self._get_library_pathes()
+
+        config =  str(self.settings.build_type).lower()
+        config += " shared" if self.options.shared else " static"
+
+        qmake_options =  f' -r "{path.join(self.source_folder, "qbs.pro")}"'
+        qmake_options += f' "QMAKE_LIBDIR={library_path}"'
+        qmake_options += f' "CONFIG-=debug_and_release debug_and_release_target debug release"'
+        qmake_options += f' "CONFIG+={config}"'
+        qmake_options += f' QMAKE_TARGET.arch={self.settings.arch}'
+        self.run(f"qmake {qmake_options}", run_environment=True)
+
+        if self.settings.os == "Windows":
+            make="jom"
+        else:
+            make="make"
+        self.run(f"{make} -j{tools.cpu_count()}", run_environment=True)
+
+    def package(self):
+        self.copy("qbs", src="bin", dst="bin")
+        self.copy("qbs.exe", src="bin", dst="bin")
+        self.copy("qbs-*", src="bin", dst="bin")
+        self.copy("*", src="share", dst="share")
+        self.copy("*", src="lib", dst="lib")
+
+    def package_id(self):
+        if self.settings.build_type == "Debug":
+            compatible_pkg = self.info.clone()
+            compatible_pkg.settings.build_type = "Release"
+            self.compatible_packages.append(compatible_pkg)
+
+    def _add_path_to_libdir(self, libdir):
+        self.output.info(f"Appending LD_LIBRARY_PATH environment variable: {libdir}")
+        if self.settings.os == "Windows":
+            self.env_info.PATH.append(libdir)
+        elif self.settings.os == "Macos":
+            self.env_info.DYLD_LIBRARY_PATH.append(libdir)
+        else:
+            self.env_info.LD_LIBRARY_PATH.append(libdir)
+
+    def package_info(self):
+        bindir = path.join(self.package_folder, "bin")
+        self.output.info(f"Appending PATH environment variable: {bindir}")
+        self.env_info.PATH.append(bindir)
+
+        if self.options.shared:
```

Comment:
It's looks like Qt isn't getting bundled in qbs package, that isn't good. AFAIK there is no flag to do it using qmake. There are `modules.qbsbuildconfig.useBundledQtScript:true` and `modules.qbsbuildconfig.enableBundledQt:true` flags to allow you to do this to build qbs with qbs.

Reply:
Ok, I will try to add this to qmake